lib/rmt/scc.rb

```diff
@@ -171,6 +171,7 @@ def create_product(item, root_product_id = nil, base_product = nil, recommended
   def create_service(item, product)
     product.create_service!
     item[:repositories].each do |repo_item|
+      repo_item[:mirroring_enabled] = true if repo_item[:installer_updates] and not repo_item[:mirroring_enabled]
```

Comment:
would there be any different behavior if we skip `and not repo_item[:mirroring_enabled]` here?

Reply:
Not right now but I don't know if that's going to change in the future and the installer-updates repositories will come already enabled.

Reply:
But even if they come already enabled that part of the code would make any difference?

Reply:
Well, technically no, I wanted to be sure that even if they come wrong, we set them right.
I will remove it.